gravitee-am-repository/gravitee-am-repository-jdbc/src/main/java/io/gravitee/am/repository/jdbc/management/api/model/JdbcPasswordPolicy.java

```diff
@@ -64,4 +65,8 @@ public class JdbcPasswordPolicy {
     private boolean passwordHistoryEnabled;
     @Column("old_passwords")
     private Short oldPasswords;
+    @Column("idp_ids")
+    private List<String> idpIds;
+    @Column("default_idp_ids")
```

Comment:
**discussion**: My first thought was to link the policy to the IDP into the domain settings in order to update all the link once instead of having potential sync issue where a policy maybe link to 2 idp during a short period of time;